plugins/inline.ts

```diff
@@ -200,14 +206,8 @@ export default function (userOptions?: Partial<Options>) {
               }
             }
 
-            // Width and height attributes
-            if (element.getAttribute("width")) {
-              svg.setAttribute("width", element.getAttribute("width")!);
-            }
-
-            if (element.getAttribute("height")) {
-              svg.setAttribute("height", element.getAttribute("height")!);
-            }
+            migrateAttributes(element, svg, ["id", "class", "width", "height"]);
+            svg.removeAttribute("src");
```

Comment:
Is this really necessary? SVG code shouldn't have a `src` attribute.

Reply:
Oops no, good spot. Just a remnant from the previous iteration. I can delete that tomorrow (or happy for you to do so)

Reply:
Okay, understood. I just fixed it. Thanks!